### PR TITLE
Auto update grafana go sdk after generation

### PR DESCRIPTION
**Pull request by academo**
**What this PR does / why we need it**:

We want newly generated plugins to always use the latest grafana go sdk. Currently the version is hardcoded in the generator templates. With this custom action we run `go get [grafana-sdk]` in the generated code upon generation which updates the go sdk to its latest version.

This pr implements the logic to run the update when the enviroment is ready to do so.

**Which issue(s) this PR fixes**:

Closes https://github.com/grafana/plugin-tools/issues/86

**Special notes for your reviewer**:

### CODE REVIEW

File: packages/create-plugin/src/commands/generate-actions/print-success-message.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { displayAsMarkdown } from '../../utils/utils.console';
+import { normalizeId } from '../../utils/utils.handlebars';
+import { CliArgs } from '../types';
+
+export function printGenerateSuccessMessage(answers: CliArgs) {
```
Comment (academo) [inline packages/create-plugin/src/commands/generate-actions/print-success-message.ts:5]: moving this out of the generate plop file

File: packages/create-plugin/src/commands/types.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { PLUGIN_TYPES } from '../constants';
```
Comment (academo) [inline packages/create-plugin/src/commands/types.ts:1]: moving this out of the generate plop file

File: packages/create-plugin/src/utils/utils.path.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import path from 'path';
```
Comment (academo) [inline packages/create-plugin/src/utils/utils.path.ts:1]: moving this out of the generate plop file

**Review by marefr — APPROVED** [review]
LGTM. Haven't run it though

**Review by jackw — APPROVED** [review]
LGTM! 🚀 

FYI - I've set the labels to release this as minor.

**Pull request merged**